pkg/users/local.js

```diff
@@ -733,12 +733,23 @@ PageAccount.prototype = {
            });
     },
 
-    get_locked: function() {
+    get_locked: function(desired_lock_state) {
```

Comment:
It seems backwards to be using a function called ```get_locked()``` to change state and execute commands. Shouldn't the desired_lock_state logic be elsewhere?